### ISSUE TITLE
Hide the expression column for Fly-Canto

### ISSUE THREAD

**Issue by kimrutherford** [pombase/canto#1810]
There is no expression information to show in Fly-Canto so we should add an option to hide the expression column in genotype tables.

**Comment by kimrutherford** [comment]
This change is done but not merged into the master branch yet.  @jseager7 is doing some work on the genotype management page and it's easier to wait for that to be done before merging.

I've added a new configuration option to control the expression. Soon we'll be able to add this to `canto_deploy.yaml` to hide the expression everywhere in the interface:

```
alleles_have_expression: false
```

**Comment by jseager7** [comment]
It looks like we've got multiple issues that can't be merged to `master` because they're blocked by my refactoring. However, the changes to the genotype management page are pretty much finished &ndash; although I might end up with a few merge conflicts when I try to merge my changes now.

I'm not sure how long the metagenotype page refactoring will take, but if necessary I could allow us to merge the changes to the genotype page sooner by re-adding the code for the 'old' organism selector, while keeping the new one around as well, and pointing the metagenotype page at the old organism selector and the genotype page at the new one. The metagenotype views are almost completely isolated from FlyBase anyway, so I could then gradually refactor the metagenotype page to replace the old organism selector with the new one.

**Comment by kimrutherford** [comment]
>> Soon we'll be able to add this to canto_deploy.yaml to hide the expression everywhere in the interface:
> `alleles_have_expression: false`

Sorry, that should be:

`alleles_have_expression: 0`

The change is now in the master Git branch.

**Comment by kimrutherford** [comment]
I've fixed the genotype edit page to obey that new setting.  I had forgotten about it.
